### PR TITLE
Updated documentation on how to move from single to multi master

### PR DESCRIPTION
**Pull request by mccare**
The steps to move from single to multi master are much easier with etcd-manager. Updated the documentation for the single-to-multimaster for the case of etcd-manager. A rollback strategy is missing. 

Outline for rollback could be: remove autoscaling groups, delete ETCD EBS volumes, revert changes made with kops edit to the cluster and kops update the cluster. Then restore the etcd-manager with etcd-manager-ctl. Since I did not test this (or only for a normal restore) I did not feel comfortable to write it down.

### CODE REVIEW

**Comment by rifelpet** [comment]
Thanks for the improvements!
/ok-to-test
/area documentation

File: docs/single-to-multi-master.md [hunk 1]
```diff
@@ -93,7 +150,10 @@ $ kops create ig master-eu-west-1c --subnet eu-west-1c --role Master
 ```
 
  * ``maxSize`` and ``minSize`` should be **0**,
+ * if you are using etcd-manager, you just can leave the `maxSize` and `minSize` at **1**.
  * only one zone should be listed.
+ * adjust the machineType
+ * adjust the image to the OS you are using
 
 ### c - Reference the new masters in your cluster configuration
```
[the remark this answers is not in the document]
Comment (bskim45) [reply docs/single-to-multi-master.md]: @mccare Can you look into this?

**Comment by mccare** [comment]
Fixed the wrong numbering and squashed and rebased the PR... Sorry for the delay.

**Pull request merged**